### DISCUSSION TITLE
Future of park-ui

### DISCUSSION THREAD

**Discussion by DawidWraga** [cschroeter/park-ui#62] [General]
It's great to see all the recent improvements to park-ui, thank you for all your time and dedication to this project.

I'm curious to better understand what is the future direction, it would be great if park-ui has a canny.io page similar to ark and panda. Alternatively, some sort of public roadmap would be really appreciated as it helps me to avoid creating components that are on the way. 

Eg I wasted time on making the card component right before it was released in park. I was going to make my own file upload component but now I see Ark is planning to implement it I will wait instead.

Thanks

**Comment by cschroeter** [comment]
Thanks for bringing this to my attention. I think having a canny board would help a lot. Here are the next steps from the top of my head:

- Improve Tailwind Plugin
- Improve documentation
- Prepare for Ark UI 1.0
- Major release (1.0) for  `@park-ui/panda-presets` and `@park-ui/tailwind-plugin`
